config/config.exs

```diff
@@ -0,0 +1,52 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+import Config
+
+config :pydis_site,
+  ecto_repos: [PydisSite.Repo]
+
+# Configures the endpoint
+config :pydis_site, PydisSiteWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: "11kMSCcgWANxMYQwHp+zCA0IwX9+rRDfVe1SNPhZDk11p+KjuvgR0B40ltDgw7M9",
```

Comment:
Good idea putting production secrets in the source code, saves needing to mess with environment variables when deploying 👍 💯